main.py

```diff
@@ -211,21 +229,32 @@ def _get_owners_id(owners):
 
 def _get_owners(client, project):
     users = []
-    project_id = project.get('projectId')
-    iamPolicy = client.projects().getIamPolicy(resource=project_id, body={}).execute()
+    project_name = project.get('name')
+    iamPolicy = client.projects().getIamPolicy(resource=project_name, body={}).execute()
     bindings = iamPolicy.get('bindings', [])
     owners = list(filter(filter_owners, bindings))
     if not owners:
-        logger.debug('No owners found for Project %s.', project_id)
+        logger.debug('No owners found for Project %s.', project_name)
     else:
         members = owners[0].get('members')
         users = list(filter(filter_users, members))
         if not users:
-            logger.debug('No owner is a user for Project %s.', project_id)
+            logger.debug('No owner is a user for Project %s.', project_name)
     users = set([user.strip('user:') for user in users])
     return list(users)
 
 
+def _get_organization(client, project):
+    client = client = discovery.build("cloudresourcemanager", "v1")
+    projectId = project.get('projectId')
+    ancestry_request = client.projects().getAncestry(projectId=projectId, body=None)
+    ancestry_response=ancestry_request.execute()
+    for resourceId in ancestry_response['ancestor']:
+        if resourceId['resourceId']['type'] == 'organization':
```

Comment:
Handle the case where the Project is not in an Org.

![image](https://user-images.githubusercontent.com/1938355/146971014-e3ab6561-4ca3-4748-9fcd-d506391c88f3.png)

Reply:
- 749efaa